src/IECoreDelight/Renderer.cpp

```diff
@@ -656,7 +656,10 @@ class DelightAttributes : public IECoreScenePreview::Renderer::AttributesInterfa
 				}
 				else if( boost::starts_with( m.first.string(), "user:" ) )
 				{
-					msg( Msg::Warning, "DelightRenderer", fmt::format( "User attribute \"{}\" not supported", m.first.string() ) );
+					if( const Data *d = reportedCast<const IECore::Data>( m.second.get(), "attribute", m.first ) )
+					{
+						params.add( m.first.c_str() + 5, d, true );
```

Comment:
Thanks for the update! I think ideally we would keep the `user:` prefix on the attribute, so that people can't accidentally overwrite built-in attributes with it. Is that possible, or does 3Delight get upset with the `:`?

Reply:
Completely makes sense. There are no 3Delight issues with having attributes with `:`, so I updated the code not to strip the `user:` prefix.

Reply:
I apologize, I forgot to update the Changes.md file to include info that both dl: and user: Gaffer attributes are used to generate NSI shaderattributes. This has been fixed now and Changes.md has been updated with the correct info.